### PR TITLE
Add KPMP usage to cl.md

### PR DESCRIPTION
**Pull request by deepakunni3**
This PR is a continuation of https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2464 where the KPMP usage is structured and added to the `usages` section of the metadata.

@yongqunh Would be great to get your feedback before this is merged in.

### CODE REVIEW

**Comment by yongqunh** [comment]
Agreed.

File: ontology/cl.md [hunk 1]
```diff
@@ -115,6 +115,15 @@ usages:
     url: http://fantom.gsc.riken.jp/5/sstar/CL:0000540
   type: annotation
   user: http://fantom5-collaboration.gsc.riken.jp/
+- description: The Kidney Precision Medicine Project (KPMP) makes use of the Kidney Tissue Atlas Ontology (KTAO) which imports terms from CL for defining kidney-related terms and their relations.
+  examples:
+  - description: The Kidney Tissue Atlas Ontology (KTAO) imports terms from CL to define kidney-related cell types, cell states, gene markers, diseases and their relations
+    url: https://ontobee.org/ontology/KTAO?iri=http://purl.obolibrary.org/obo/CL_1000497
```
Comment (matentzn) [inline ontology/cl.md:121]: We do not generally like this kind of evidence. If you ask Chris, he would want the evidence to be a URL to a scientific database resource, not an ontology import. We typically avoid usage declarations that involve "used by another ontology" as we understand that this is "a given". Is there a way to proof CL useage in the KMPM project outside providing a link to the ontology?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: ontology/cl.md (hunk 1 above)
Comment (deepakunni3) [reply ontology/cl.md:121]: Thank you for the feedback @matentzn :)

@yongqunh Would you be able to provide a more appropriate usage of CL in KPMP project?

Comment (addiehl) [reply ontology/cl.md:121]: Figure 7  of https://www.biorxiv.org/content/10.1101/2024.04.01.587658v1.full.pdf

Comment (matentzn) [reply ontology/cl.md:121]: What we would need to be by the book is a link to the "Kidney Tissue Atlas" which shows a CL term in use. Neither a link to an ontology and a link to a paper fulfills that criterion.

That said, this is just my own annoying record spinning. The verbatim of the obofoundry principle allows the use of papers as evidence (albeit, be spirit, not exactly pre-prints). So you can except a doi to that paper I guess.

**Comment by yongqunh** [comment]
Here is the KPMP paper that cites Cell Ontology: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8012202/

**Comment by nlharris** [comment]
What's the status of this?

**Comment by deepakunni3** [comment]
Closing this PR since listing the publication that cites CL is insufficient.

Will revisit this at a later date, if needed.

**Pull request closed without merging**